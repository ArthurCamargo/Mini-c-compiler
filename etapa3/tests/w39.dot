digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 [label="a"];
n3;
n3 -> n4;
n3 -> n5;
n4;
n4 -> n6;
n4 -> n7;
n6;
n6 [label="b"];
n7;
n7 -> n8;
n7 -> n9;
n8;
n8 -> n10;
n8 -> n11;
n10;
n10 [label="c"];
n11;
n11 [label="d"];
n8 [label="*"];
n9;
n9 [label="e"];
n7 [label="/"];
n4 [label="+"];
n5;
n5 [label="f"];
n3 [label="-"];
n1 [label="="];
n0 [label="f"];

}