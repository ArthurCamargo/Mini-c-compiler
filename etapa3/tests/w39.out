0x55c578fed350
0x55c578fed350, 0x55c578feda20
0x55c578feda20
0x55c578feda20, 0x55c578fed660
0x55c578feda20, 0x55c578fed9d0
0x55c578fed660
0x55c578fed660 [label="a"];
0x55c578fed9d0
0x55c578fed9d0, 0x55c578fed910
0x55c578fed9d0, 0x55c578fed980
0x55c578fed910
0x55c578fed910, 0x55c578fed6d0
0x55c578fed910, 0x55c578fed8c0
0x55c578fed6d0
0x55c578fed6d0 [label="b"];
0x55c578fed8c0
0x55c578fed8c0, 0x55c578fed800
0x55c578fed8c0, 0x55c578fed870
0x55c578fed800
0x55c578fed800, 0x55c578fed740
0x55c578fed800, 0x55c578fed7b0
0x55c578fed740
0x55c578fed740 [label="c"];
0x55c578fed7b0
0x55c578fed7b0 [label="d"];
0x55c578fed800 [label="*"];
0x55c578fed870
0x55c578fed870 [label="e"];
0x55c578fed8c0 [label="/"];
0x55c578fed910 [label="+"];
0x55c578fed980
0x55c578fed980 [label="f"];
0x55c578fed9d0 [label="-"];
0x55c578feda20 [label="="];
0x55c578fed350 [label="f"];
